sdk/ai/azure-ai-projects/samples/agents/async_samples/sample_agents_azure_functions_async.py

```diff
@@ -0,0 +1,106 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import asyncio
+
+"""
+FILE: sample_agents_azure_functions_async.py
+ 
+DESCRIPTION:
+    This sample demonstrates how to use azure function agent operations from
```

Comment:
Could you please add link to quickstart or doc which describes how user can create Azure function and get the required parameters for tool construction?

Reply:
Added.